### ISSUE TITLE
retrace-server with ftp task from 'manager' page now errors out with "Please use VRA format for kernel version (e.g. 2.6.32-287.el6.x86_64)"

### ISSUE THREAD

**Issue by DaveWysochanskiRH** [abrt/retrace-server#312]
Guessing it is one of the recent patches to 'manager' but not sure yet, drilling down and guess:
53fd9b59975d9
or
eb2e2d1861521

```
$ git blame src/manager.wsgi | grep -B 10 "Please use VRA format for kernel version (e.g. 2.6.32-287.el6.x86_64)"
53fd9b59975d9 (Martin Kutlak       2020-03-09 17:02:56 +0100 163)         debug = "debug" in GET
91a6d6cd772cf (Michal Toman        2014-06-11 14:34:15 +0200 164)         kernelver = None
91a6d6cd772cf (Michal Toman        2014-06-11 14:34:15 +0200 165)         arch = None
53fd9b59975d9 (Martin Kutlak       2020-03-09 17:02:56 +0100 166)         if "kernelver" in GET:
94da5fcb4a068 (Michal Toman        2013-02-26 11:10:53 +0100 167)             try:
53fd9b59975d9 (Martin Kutlak       2020-03-09 17:02:56 +0100 168)                 kernelver = KernelVer(GET["kernelver"])
00f15b055c0df (Michal Toman        2013-06-26 15:13:36 +0200 169)                 if kernelver.arch is None:
00f15b055c0df (Michal Toman        2013-06-26 15:13:36 +0200 170)                     raise Exception
94da5fcb4a068 (Michal Toman        2013-02-26 11:10:53 +0100 171)             except Exception as ex:
e17f5b9042aec (Jan Beran           2018-06-11 16:02:57 +0200 172)                 return response(start_response, "403 Forbidden",
e17f5b9042aec (Jan Beran           2018-06-11 16:02:57 +0200 173)                                 _("Please use VRA format for kernel version (e.g. 2.6.32-287.el6.x86_64)"))
--
53fd9b59975d9 (Martin Kutlak       2020-03-09 17:02:56 +0100 299)         if "vra" in POST:
53fd9b59975d9 (Martin Kutlak       2020-03-09 17:02:56 +0100 300)             vra = POST["vra"]
5b1542eb0ba40 (Michal Toman        2014-02-27 13:33:59 +0100 301) 
eb2e2d1861521 (Martin Kutlak       2020-03-09 17:20:25 +0100 302)             if vra.strip():
5b1542eb0ba40 (Michal Toman        2014-02-27 13:33:59 +0100 303)                 try:
5b1542eb0ba40 (Michal Toman        2014-02-27 13:33:59 +0100 304)                     kver = KernelVer(vra)
5b1542eb0ba40 (Michal Toman        2014-02-27 13:33:59 +0100 305)                     if kver.arch is None:
5b1542eb0ba40 (Michal Toman        2014-02-27 13:33:59 +0100 306)                         raise Exception
5b1542eb0ba40 (Michal Toman        2014-02-27 13:33:59 +0100 307)                 except:
e17f5b9042aec (Jan Beran           2018-06-11 16:02:57 +0200 308)                     return response(start_response, "403 Forbidden",
e17f5b9042aec (Jan Beran           2018-06-11 16:02:57 +0200 309)                                     _("Please use VRA format for kernel version (e.g. 2.6.32-287.el6.x86_64)"
```

**Comment by DaveWysochanskiRH** [comment]
Looks due to 53fd9b5

>      elif match.group(6) and match.group(6) == "start":
>          # start
> -        get = urllib.parse.parse_qs(request.query_string)
> +        GET = request.GET
>          ftptask = False
>          try:
>              task = RetraceTask(filename)
> @@ -137,16 +138,16 @@ def application(environ, start_response):
>              except:
>                  return response(start_response, "500 Internal Server Error", _("Unable to create a new task"))
>  
> -            if "caseno" in get:
> +            if "caseno" in GET:
>                  try:
> -                    task.set_caseno(int(get["caseno"][0]))
> +                    task.set_caseno(int(GET["caseno"]))
>                  except:
>                      # caseno is invalid number - do nothing, it can be set later
>                      pass
>  
> -            if "bugzillano" in get:
> +            if "bugzillano" in GET:
>                  try:
> -                    bugzillano = list(filter(int, set(n.strip() for n in get["bugzillano"][0].
> +                    bugzillano = list(filter(int, set(n.strip() for n in GET["bugzillano"].
>                                                        replace(";", ",").split(","))))
>                      task.set_bugzillano(bugzillano)
>                  except Exception:
> @@ -156,12 +157,12 @@ def application(environ, start_response):
>          if not task.get_managed():
>              return response(start_response, "403 Forbidden", _("Task does not belong to task manager"))
>  
> -        debug = "debug" in get
> +        debug = "debug" in GET
>          kernelver = None
>          arch = None
> -        if "kernelver" in get:
> +        if "kernelver" in GET:
>              try:
> -                kernelver = KernelVer(get["kernelver"][0])
> +                kernelver = KernelVer(GET["kernelver"])
>                  if kernelver.arch is None:
>                      raise Exception
>

**Comment by ernestask** [comment]
Presumably fixed by #313.
